src/components/src/modals/cloud-components/cloud-item.spec.tsx

```diff
@@ -0,0 +1,59 @@
+// @ts-nocheck
```

Comment:
working on it. i want to make the case about colocating tests next to their source because it improves developer ergonomic and make writing tests much quicker and simpler. 
I had no problem mocking custom hooks in this setup using jest but if i move the test in the Test folder i need to mock kepler.gl/components with several side effects

Reply:
we are good to go